website/static/files/build_orca_inventory_refined.sql

```diff
@@ -4,7 +4,7 @@ SET search_path TO orca, public;
 
 
 -- Create the objects in a transaction.
-BEGIN
+BEGIN;
```

Comment:
just making sure if  adding the `;` is intentional?

Reply:
Yes, without the SQL will not run properly for this stand alone file.